db/seeds/development/course.seeds.rb

```diff
@@ -0,0 +1,12 @@
+# Seed Course and Section tables in development environment.
+#
+# bin/rails db:seed
+
+Course.create title: 'Photography I', short_title: 'P1'
+
+Section.create session: 1, period: 1, course_id: 1, name: 'S1P1-P1'
```

Comment:
You can pass the course object in `course:` and Rails will figure out the `course_id` for you.